R/html-tweak.R

```diff
@@ -83,7 +83,7 @@ tweak_tables <- function(html) {
 }
 
 prepend_class <- function(x, class = "table") {
-  if (length(x) == 0) {
```

Comment:
If this is the require fix, please write a unit test for `prepend_class()` rather than testing the building of a complete website.

Reply:
I didn’t want to change @jayhesselberth test. I’ll look how he set it up.

Reply:
Yes, I'd recommend adding a new test along those lines.

Reply:
Also this should have spaces around `%in%`